### PR TITLE
Rename sdk-development -> web5-spec

### PR DESCRIPTION
**Pull request by leordev**

### CODE REVIEW

**Review by diehuxx — APPROVED** [review]

**Review by nitro-neal — COMMENTED** [review]
so instead of web5-spec, the runner will be hosted here - 
https://github.com/TBD54566975/sdk-report-runner

File: .github/workflows/tests-ci.yml [hunk 1]
```diff
@@ -83,9 +83,9 @@ jobs:
           app-id: ${{ secrets.CICD_ROBOT_GITHUB_APP_ID }}
           private-key: ${{ secrets.CICD_ROBOT_GITHUB_APP_PRIVATE_KEY }}
           owner: TBD54566975
-          repositories: sdk-development
+          repositories: web5-spec
```
Comment (nitro-neal) [inline .github/workflows/tests-ci.yml:86]:
```suggestion
          repositories: sdk-report-runner
```

File: .github/workflows/tests-ci.yml [hunk 2]
```diff
@@ -83,9 +83,9 @@ jobs:
...
 
-      - name: Trigger sdk-development report build
+      - name: Trigger web5-spec report build
```
Comment (nitro-neal) [inline .github/workflows/tests-ci.yml:88]:
```suggestion
      - name: Trigger sdk-report-runner report build
```

File: .github/workflows/tests-ci.yml [hunk 3]
```diff
@@ -94,7 +94,7 @@ jobs:
           -H "Content-Type: application/json" \
           --fail \
           --data '{"ref": "main"}' \
-          https://api.github.com/repos/TBD54566975/sdk-development/actions/workflows/build-report.yaml/dispatches
+          https://api.github.com/repos/TBD54566975/web5-spec/actions/workflows/build-report.yaml/dispatches
```
Comment (nitro-neal) [inline .github/workflows/tests-ci.yml:97]:
```suggestion
          https://api.github.com/repos/TBD54566975/sdk-report-runner/actions/workflows/build-report.yaml/dispatches
```

**Comment by frankhinek** [comment]
In last week's Web5 Spec call, it was agreed that `sdk-development` would be renamed to `web5-spec` and the report generator / runner would be moved to its own repo.  I believe that work is still in progress but will need to be completed and then this PR updated to point to the new repo.

**Review by nitro-neal — APPROVED** [review]
update to sdk-report-runner and yea should be good to go, 

The code at sdk-report-runner may not be ready and an action may fail, but should be up and running soon

**Comment by frankhinek** [comment]
@finn-tbd IIRC you requested responsibility to migrate the report runner over to the new repo.

What's your ETA?  If its not that far out we could hold off on approving this PR until the new `sdk-report-runner` repo is ready.

**Review by frankhinek — CHANGES_REQUESTED** [review]
In last week's Web5 Spec call, it was agreed that `sdk-development` would be renamed to `web5-spec` and the report generator / runner would be moved to its own repo.  I believe that work is still in progress but will need to be completed and then this PR updated to point to the new repo.

**Comment by finn-tbd** [comment]
That was waiting on https://github.com/TBD54566975/web5-spec/pull/99, which just merged. I believe @nitro-neal is going to handle the rename now as i'm getting pulled into did:dht work

**Review by nitro-neal — APPROVED** [review]

**Review by nitro-neal — APPROVED** [review]

**Comment by leordev** [comment]
@frankhinek the changes were addressed, can you please review it again? If it looks good for you, please go ahead and merge!

**Review by frankhinek — APPROVED** [review]

**Pull request merged**